fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -76559,7 +76644,7 @@ creation_date: 2010-07-14T02:46:14Z
 id: FBbt:00100170
 name: extra LNd neuron
 def: "LNd neuron of the adult brain characterized by having a slightly larger nucleus and cytoplasmic area compared to the other LNd neurons." [FlyBase:FBrf0191052]
-comment: Rieger et al., (2006) comment that the extra LNd also stains more strongly for Period (FBgn0003068) and Timeless (FBgn0014396) compared to the other LNd neurons. Under constant light (LL) conditions, the extra LNd neuron is the only LNd neuron which shows significant Period and Timeless cycling.
+comment: Rieger et al., (2006) comment that the extra LNd also stains more strongly for Period (FBgn0003068) and Timeless (FBgn0014396) compared to the other LNd neurons. Under constant light (LL) conditions, the extra LNd neuron is the only LNd neuron which shows significant Period and Timeless cycling.\n\nThis neuron must belong to one of the other LNd subclasses. It is most likely equivalent to the ITP and CRY expressing LNd neuron, as this has the largest cell diameter among LNd neurons (Schubert et al., 2017 - FlyBase:FBrf0238313).
```

Comment:
Minor formatting issue: please avoid adding returns in text fields.  (newline appears as "\n" above.)